### PR TITLE
MIAJS-949 - Analytics events

### PR DESCRIPTION
**Pull request by simonlaustsen**

### CODE REVIEW

File: collections/_web/v3/guides/google_analytics.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+parent: guides
+layout: tutorial
+nav_weight: 1100
+published: true
+date: 2020-01-15
+---
+
+## Overview
+
+In this, we will take a look at Google Analytics.
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
In this, we will take a look at Google Analytics.

```

File: collections/_web/v3/guides/google_analytics.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Google Analytics is a free tool that gives you the opportunity to analyze data to get a deeper understanding of your customers and how they use your app. An example could be to figure out which places is hardest to find on your venue or what users typically search for when looking for a location.
+
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
```

File: collections/_web/v3/guides/google_analytics.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Get started
+
+To start seeing stats for your [standard web app](http://clients.mapsindoors.com/):
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]: Link to a demo app?

File: collections/_web/v3/guides/google_analytics.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Create or sign in to your Analytics account:
+* Go to [google.com/analytics](https://marketingplatform.google.com/about/analytics/)
+* Do one of the following:
+  * To create an account, click Start for free.
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
  * To create an account, click "Start for free".
```

File: collections/_web/v3/guides/google_analytics.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  * To sign in to your account, Click Sign in to Analytics.
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
  * To sign in to your account, Click "Sign in to Analytics".
```

File: collections/_web/v3/guides/google_analytics.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+1. [Set up a property in your Analytics account](https://support.google.com/analytics/answer/1042508). A [property](https://support.google.com/analytics/answer/2649554?hl=en&ref_topic=1009620) represents your website, and is the collection point in Analytics for the data from your site.
+1. Provide your MapsIndoors project manager with your newly created Property ID.
+
+When done you will within 24 hours start receiving stats and events in your Google Analytics Dashboard.
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
After doing this, within 24 hours you will start receiving stats and events in your Google Analytics Dashboard.
```

File: collections/_web/v3/guides/google_analytics.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+### Events overview
+
+When a Property Id is added to your MapsIndoors solution you will see the following events in Google Analytics.
```
Comment (marksdk) [inline collections/_web/v3/guides/google_analytics.md]:
```suggestion
When a `Property ID` is added to your MapsIndoors solution you will see the following events in Google Analytics.
```

File: collections/_web/v3/guides/google_analytics.md (hunk 3 above)
Comment (simonlaustsen) [reply collections/_web/v3/guides/google_analytics.md]: Link to login page yes :)

**Comment by marksdk** [comment]
@simonlaustsen Ship away!

**Pull request merged**